package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "start": "nodemon --exec \"ts-node\" ./src/server.ts",
-    "test": "export NODE_ENV=test && nyc mocha -r ts-node/register --exit ./src/test/*.test.ts --config .mocharc.json",
+    "test": "export NODE_ENV=test && nyc mocha -r ts-node/register --exit ./src/test/*Test.ts --config .mocharc.json",
```

Comment:
`.test` or `.spec` are the standard way of naming tests. Why did you rename tests? Is there an issue you were facing?

Reply:
On it. 

> `.test` or `.spec` are the standard way of naming tests. Why did you rename tests? Is there an issue you were facing?
Just cosmetics. Reversing it to my previous naming hehe